aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -588,5 +602,75 @@ TORCH_IMPL_FUNC(_convert_indices_from_csr_to_coo_structured_cpu) (
   }
 }
 
+Tensor& linalg_solve_sparse_csr_out(const Tensor& input, const Tensor& other, Tensor& result) {
+  if (at::globalContext().hasCUDA()) {
+    TORCH_INTERNAL_ASSERT(input.is_sparse_csr());
+
+    c10::MaybeOwned<Tensor> other_ = other.expect_contiguous();
+    c10::MaybeOwned<Tensor> result_ = result.expect_contiguous();
+
+    if (other.ndimension() > 1) {
+      TORCH_CHECK(false, "NotImplementedError: other tensor with dimension > 1 is not implemented yet.");
+    }
```

Comment:
This check prevents passing tensors with shape `(n, 1)`.
```suggestion
    if (other.ndimension() > 1) {
      TORCH_CHECK(other.size(-1) == 1, "NotImplementedError: multiple vector case stored in 'other' tensor is not implemented yet.");
    }
```